west.yml

```
@@ -101,7 +101,7 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: 6ee81029d57d77ee7cbd9e9b7f1740f347e4dce2
+      revision: 689d669c8d5120e173905fa40009ebbb8b1e54c0
```

```suggestion
      revision:  pull/330/head
```
Need to point to the PR for now, and update after the nrfxlib PR is merged.

---

Sure, my bad :)